docs/getting_started/scaleway.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with kops on Scaleway
+
+**WARNING**: Scaleway support on kOps is currently in **alpha**, which means that it is in the early stages of development and subject to change, please use with caution.
+
+## Requirements
+
+* [kops version >= 1.18 installed](../install.md)
+* [kubectl installed](../install.md)
+* [Setup your SSH key](https://www.scaleway.com/en/docs/configure-new-ssh-key/) : id_rsa and id_ed25519 keys are supported
+* [Scaleway credentials](https://www.scaleway.com/en/docs/generate-api-keys/) : you will need at least an access key, a secret key and a project ID.
+* [S3 bucket and its credentials](https://www.scaleway.com/en/docs/storage/object/quickstart/) : the bucket's credentials may differ from the one used for provisioning the resources needed by the cluster. If you use a Scaleway bucket, you will need to prefix the bucket's name with `scw://` in the `KOPS_STATE_STORE` environment variable. For more information about buckets, see [here](../state.md)
+
+### Optional
+* If you wish to use your own domain name for your cluster, you'll need a [Scaleway managed domain](https://www.scaleway.com/en/docs/network/domains-and-dns/quickstart). If not, you can always use the [gossip DNS](https://kops.sigs.k8s.io/gossip/) by naming your cluster with the suffix `.k8s.local`.
+
+## Environment Variables
+
+It is important to set the following [environment variables](https://github.com/scaleway/scaleway-sdk-go/blob/master/scw/README.md):
+```bash
+export SCW_ACCESS_KEY="my-access-key"
+export SCW_SECRET_KEY="my-secret-key"
+export SCW_DEFAULT_PROJECT_ID="my-project-id"
+export SCW_DEFAULT_REGION="fr-par"
+export SCW_DEFAULT_ZONE="fr-par-1"
```

Comment:
Why do these have to be defined in the environment?

Reply:
In case you are not specifying the zone from the command line, the SDK will use those environment variables to configure the locality of the nodes.

Reply:
@remyleone I should indeed try to get rid of these variables because `zones` is required in the command line when creating a cluster. It was simpler to pass an environment variable because sometimes I have to call the SDK from functions that do not have access to the cluster spec (where the zone is stored) but I can try again to find a way.

Reply:
I think they could be defined in either a configuration file, in an environment variable or a command line argument. viper will handle those cases :)

Reply:
fixed in #14972